index.js

```diff
@@ -1,7 +1,10 @@
 import { ChatOpenAI } from 'langchain/chat_models/openai'
 import { initializeAgentExecutorWithOptions } from 'langchain/agents'
 import { createInterface } from 'readline'
-import { GoogleCalendarTool } from './src/google_calendar_tool.js'
+import {
+  GoogleCalendarCreateTool,
+  GoogleCalendarViewTool
+} from './src/google_calendar_tool.js'
```

Comment:
Maybe it's out of scope for this PR but would it make sense to have the contents of this file in something like a `bin.js` tailored to running from scripts and have the `main` package entry file export both of these?

So it's possible to do `npm run start` and enter prompts that will run the appropriate tool or tools, and it's also possible to do `import { GoogleCalendarViewTool } from 'langchain-google-calendar'` for use cases like integrating this functionality into a company bot?

Reply:
I think this covers similar ground to https://github.com/nearform/langchain-google-calendar/issues/35